modules/ding_gatewayf/modules/ding_gatewayf_registration/default_information_text.html

```diff
@@ -0,0 +1,58 @@
+<h2><span class="change-me">Danevangs Kommunes Bibliotekers</span> behandling af personoplysninger om dig</h2>
```

Comment:
`<span class="change-me">` - could we replace this with something like .ding-gateway--temporary-text`?

Could we replace this with something better/more relevant like the site name from the drupal configuration?

Reply:
Class names will be changed.